### PR TITLE
Improved Plex support

### PR DESCRIPTION
**Pull request by psyko-gh**

### CODE REVIEW

File: apps/backend/Cargo.toml [hunk 1]
```diff
@@ -107,6 +107,7 @@ tracing = { version = "0.1.37", features = ["attributes"] }
 tracing-appender = "0.2.2"
 tracing-subscriber = "0.3.17"
 uuid = "1.4.1"
+regex = "1"
```
Comment (IgnisDa) [inline apps/backend/Cargo.toml:110]: Can you put this in the correct alphabetic place?

File: apps/backend/src/miscellaneous/resolver.rs [hunk 2]
```diff
@@ -186,6 +186,7 @@ struct CreateUserNotificationPlatformInput {
 #[derive(Debug, Serialize, Deserialize, InputObject, Clone)]
 struct CreateUserSinkIntegrationInput {
     lot: UserSinkIntegrationSettingKind,
+    username: Option<String>,
```
Comment (IgnisDa) [inline apps/backend/src/miscellaneous/resolver.rs:189]: This should be a part of `UserSinkIntegrationSetting`.

**Review by IgnisDa — COMMENTED** [review]
@psyko-gh LGTM. Is this ready to be merged or do you want to make any other changes?

**Comment by psyko-gh** [comment]
> Was the problem with the incorrect TMDB ID for shows solved?

No, as we discussed I created the PR so you can work on it :) I only added the parsing of the TV show name from the payload.
That's also why I didn't mentionned it in the docs (nor removed the experimental warning for Plex). So unless you work on it, commit https://github.com/IgnisDa/ryot/pull/342/commits/467e35afb097cfe8ff899dbeba365da69e4d14c6 is not relevant

**Comment by IgnisDa** [comment]
Alright I will take a look at it.

**Pull request merged**